src/views/components/timeline/Timeline.web.js

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react';
+import { shape, string, number } from 'prop-types';
+import * as ReactTimeline from 'react-timeline-scribble';
+
+const   Timeline  = ( props ) => {
+  const renderEvents = () => {
+    const { data } = props;
+
+    return data.map( dd => (
+      <ReactTimeline.Event
+        interval={dd.date}
+        title={dd.title}
+        key={dd.key}
+      />
+    ));
+  };
+
+  return (
+    <Fragment>
+      <ReactTimeline.Timeline>
+        {renderEvents()}
```

Comment:
Are we able to inline this block of code here instead of calling a function? Saves the need to create & call another function each render and reduces the complexity of your code. Thanks!